rpm/generic/zfs.spec.in

```diff
@@ -87,17 +87,28 @@
 %define __python_pkg_version      2
 %define __python_cffi_pkg         python-cffi
 %define __python_setuptools_pkg   python-setuptools
+%define __python_sitelib          %{python_sitelib}
```

Comment:
just do this:

`%define __python_sitelib %(%{__python} -Ic "from distutils.sysconfig import get_python_lib; print(get_python_lib())")`

Reply:
This ```%{__python} -Ic``` only works for python3.
However I can use the selected __python value to cleanup and make it generic.

Reply:
`-I` is not required. You can swap it for `-Es` instead.